cosmoshub/assetlist.json

```diff
@@ -731,7 +731,56 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.svg"
       }
     },
-    {
+      {
+      "description": "Join Matcha Club - use our ecosystem for for free.\nNFT builder, utility apps. Cooking non-stop!",
+      "extended_description": "Team of proved builders. Verified fams. Join us!",
+      "denom_units": [
+        {
+          "denom": "ibc/B52C691E5C17CF2A0039B824931C292898D4E9A22D085EB75B89A766B96DE81E",
+          "exponent": 0
+        },
+        {
+          "denom": "MTC",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/B52C691E5C17CF2A0039B824931C292898D4E9A22D085EB75B89A766B96DE81E",
+      "name": "Matcha Club",
+      "display": "MTC",
+      "symbol": "MTC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/MTC.png"
+      },
+      "socials": {
+        "website": "https://builder.crmh.xyz",
+        "twitter": "https://x.com/crypmatcha"
+      },
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/mtc",
+            "channel_id": "channel-75"
```

Comment:
Trace of cosmoshub, ibc/B52C691E5C17CF2A0039B824931C292898D4E9A22D085EB75B89A766B96DE81E makes reference to IBC channels not registered.
channel-7[5](https://github.com/cosmos/chain-registry/actions/runs/15475016331/job/43582397771?pr=6320#step:4:6), channel-0
channel-0, channel-141

Reply:
I don't think you've entered the right channels